pymkv/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 # set the version number within the package using importlib
 try:
-    __version__: str | None = importlib.metadata.version(__name__)
+    __version__: str | None = importlib.metadata.version("pymkv2")
```

Comment:
Because the module is `pymkv` but the package is `pymkv2` it could not find it in the environment.

Reply:
FYI @GitBib